<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirResourceImportIntroWidget</class>
 <widget class="QWidget" name="DirResourceImportIntroWidget">
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="warning1">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string comment="@info">KAlarm no longer supports calendar directory resources. In order to continue using the contents of the directory resource below, you must import it into a KAlarm calendar file resource.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="warning2">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string comment="@info">KAlarm no longer supports calendar directory resources. In order to continue using the contents of the directory resource below, you must import each of its alarm types into separate KAlarm calendar file resources.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label1">
     <property name="text">
      <string comment="@label:textbox">Display name:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="dirNameLabel">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label2">
     <property name="text">
      <string comment="@label:textbox">Calendar directory:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="dirPathLabel">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label3">
     <property name="text">
      <string comment="@label:textbox">Alarm types:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="dirTypesLabel">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="note">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="text">
      <string comment="@info">If you prefer, you can import this calendar directory resource later, by using the Import Alarms menu option to import the calendar files contained within it.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets/>
 <resources/>
 <connections/>
</ui>
